build.ps1

```diff
@@ -1,6 +1,13 @@
 Set-StrictMode -Version Latest
 $ErrorActionPreference = 'Stop'
 
+function Invoke-CheckExitCode([scriptblock]$ScriptBlock) {
+    & $ScriptBlock
+    if ($LASTEXITCODE -ne 0) {

```

Comment:
I usually see this as `$?`, and `if (!$?) ...`

Reply:
`$?` indicates if the previous command succeeded or not, `$LASTEXITCODE` is the actual exit code.